src/PagesComponents/PersonLink.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Person } from '../types';
+
+type Props = {
+  person: Person;
+  selectedPersonSlug: string | undefined;
+};
+
+export const PersonLink: React.FC<Props> = ({ person, selectedPersonSlug }) => {
+  const isSelectedPerson = selectedPersonSlug === person.slug;
+
+  return (
+    <tr
+      data-cy="person"
+      className={classNames(
+        { 'has-background-warning': isSelectedPerson },
+        { '': !isSelectedPerson },
+      )}
+    >
+      <td>
+        <a
+          className={person.sex === 'f' ? 'has-text-danger' : ''}
+          href={`#/people/${person.slug}`}
```

Comment:
```suggestion
          href={`/people/${person.slug}`}
```
Router itself will manage this hash.